src/org/usfirst/frc4904/autonly/CenterSwitchDistance.java

```diff
@@ -0,0 +1,34 @@
+package org.usfirst.frc4904.autonly;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.commands.RunIfElse;
+import org.usfirst.frc4904.standard.commands.chassis.ChassisMoveDistance;
+import org.usfirst.frc4904.standard.commands.chassis.ChassisTurn;
+
+public class CenterSwitchDistance extends Strategy {
+	public static final double DISTANCE_APPROACH_MID = -1; // TODO: real values
+	public static final double DISTANCE_MID_SWITCH = -1;
+	public static final double DEGREES_TURN_LEFT = -1;
+	public static final double DEGREES_TURN_RIGHT = -1;
+
+	public CenterSwitchDistance() {
+		addSequential(
+			new ChassisMoveDistance(RobotMap.Component.chassis, RobotMap.Metrics.Wheel.TICKS_PER_INCH * DISTANCE_APPROACH_MID,
```

Comment:
I'm like 90% sure that our formatter doesn't opt for a newline after the parenthesis on a method call. Please fix.

Reply:
Apparently I was wrong which is worse.